### DISCUSSION TITLE
Steam emu

### DISCUSSION THREAD

**Discussion by drbrakbek** [m4dEngi/RemotePlayWhatever#55] [Q&A]
I assume it won't work with steam emulators because they probably dont have the remote play stuff integrated right? (talking about goldberg_emulator found on gitlab for offline LAN purpouses. I often host LAN partys)

**Comment by TuTuRSiwel** [comment]
I didn't have the chance to try it since it broke with last steam update, but remote play together by it self works 100% with emulators. I tried dolphin and cemu(using opengl for rendering (vulkan is breaking the overlay)). 
In the meantime there is a work around to play any application in remote play together here is a tutorial : https://www.youtube.com/watch?v=CEIKMCFsUfI&ab_channel=Rucati

**Reply by m4dEngi** [reply]
https://github.com/m4dEngi/RemotePlayWhatever/issues/7#issuecomment-1042865156 here's an easier method.

**Reply by m4dEngi** [reply]
It mentions spacewar, but spacewar doesn't work anymore, so you can use AppID of any game with full controller support.

**Reply by drbrakbek** [reply]
i'm aware of using other steamappids but would love a 100% offline LAN solution not needing even steam itself. That was why I was thingking about goldbergemu since it provides steam emulation and LAN fnctionality. It does not include remote play so i guess it won't work.

**Comment by Lolipop2511** [comment]
Its don't working

**Reply by m4dEngi** [reply]
Try the latest version.

**Reply by Krazon** [reply]
> Try the latest version.

Latest version doesn't work since there has been another steam update. I'm no programmer at all but is there no way to prevent this from happening with every steam update? It's not like they update steam remote play together that often. Wish they would just enable it for every game :(
